pytradfri/VERSION

```diff
@@ -1 +1 @@
-8.0.1
+8.1.0
```

Comment:
Technically, removing support for Python 3.7 is a breaking change. Should we bump major version?

Reply:
Let’s be prudent, I’ll bump major.